.gitignore

```
@@ -18,3 +18,6 @@ DataPlotly/i18n/
 
 # Vscode
 .vscode/*
+
+# Test
+DataPlotly/test/*.qgs~
```

```suggestion
*.qgs~
```

let's ignore all the project backup files